salsah/src/public/vendor/ckeditor-4.7.1-Editor/config.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+CKEDITOR.editorConfig = function( config ) {
```

Comment:
I have the feeling that I have already asked you that ... 

But I do not remember your answer :-(